pkg/filesystem/file_info.go

```diff
@@ -15,6 +15,8 @@ const (
 	FileTypeDirectory
 	// FileTypeSymlink means the file is a symbolic link.
 	FileTypeSymlink
+	// FileTypeCharacterDevice means the file is a character device
```

Comment:
I suspect that this part of the change isn't really needed, right? I guess none of the code in Buildbarn ever needs to distinguish between FileTypeCharacterDevice and FileTypeOther.

Reply:
In the patch which uses this functionality (https://github.com/buildbarn/bb-remote-execution/pull/52) I make use of this distinction to skip uploading character devices without failing, whilst still allowing the upload to fail if `FileTypeOther` appears. https://github.com/buildbarn/bb-remote-execution/pull/52/files#diff-48b3c6a7d58dce08bdf7f742b515e206